libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/tasks/TasksProvider.kt

```diff
@@ -55,6 +53,12 @@ internal fun <T : Task> Project.registerTask(
 }
 
 
+internal fun Project.locateTaskByName(name: String): TaskProvider<*>? = try {
```

Comment:
nice, but maybe `namedOrNull` would be a better name. It would show clearly that we use the TCA API.

Reply:
Can we also rename `locateTask` on line 61-66 to `namedOrNull`. We loose the notion of overload by changing the method name.

Reply:
We could even define here a extensions for `project.tasks.named` to `Project.named()` so that the API used for tasks all point to this file and make it easier to read.